evap/evaluation/models.py

```diff
@@ -1028,19 +1028,23 @@ def __render_string(cls, text, dictionary):
         return Template(text).render(Context(dictionary, autoescape=False))
 
     @classmethod
-    def send_to_users_in_courses(self, template, courses, recipient_groups):
+    def send_to_users_in_courses(self, template, courses, recipient_groups, cc):
         user_course_map = {}
         for course in courses:
             responsible = course.responsible_contributor
-            for user in self.recipient_list_for_course(course, recipient_groups):
-                if user not in responsible.cc_users.all() and user not in responsible.delegates.all():
-                    user_course_map.setdefault(user, []).append(course)
+            all_recipients = self.recipient_list_for_course(course, recipient_groups)
+            cc_recipients = []
+            if responsible in all_recipients and cc == True:
+                cc_recipients.extend(responsible.cc_users.all())
+                cc_recipients.extend(responsible.delegates.all())
+            recipients = [recipient for recipient in all_recipients if recipient not in cc_recipients]
```

Comment:
this code removes users from the recipient list if they'll get the email anyways because they are cc'ed in the email to a course's responsible person so they don't get the email twice, right? and this will break if they are in the recipient list for another course as well, right? and we don't care because that won't happen often enough in production, right?

if yes, could you add some comment along those lines to the code? i needed quite a while right now to figure that out.